eng/code-quality-reports/src/main/resources/revapi/revapi.json

```diff
@@ -633,6 +633,12 @@
           "old": "class com.azure.resourcemanager.sql.models.UserIdentity",
           "new": "class com.azure.resourcemanager.sql.models.UserIdentity",
           "justification": "Customer unlikely to subclass this class."
+        },
+        {
+          "ignore": true,
+          "code": "java.method.removed",
+          "old": "method com.azure.resourcemanager.compute.models.WindowsConfiguration com.azure.resourcemanager.compute.models.WindowsConfiguration::withEnableVMAgentPlatformUpdates(java.lang.Boolean)",
+          "justification": "Fix method removed."
```

Comment:
`Service changed the property to readOnly.`

https://github.com/Azure/azure-rest-api-specs/pull/29575#discussion_r1653671646
It's been approved.

Also mention it in `Breaking Changes` section.

Reply:
Fixed in the new version.